log-analytics.tf

```
@@ -0,0 +1,8 @@
+resource "azurerm_log_analytics_workspace" "migration-analytics" {
+  count               = local.is_migration_environment ? 1 : 0
+  name                = "migration-analytics-workspace"
```

```suggestion
  name                = "darts-migration-analytics-workspace"
```

Just makes it easier to identify in the portal whose LAW this is :)